packages/chat-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -64,7 +64,7 @@ export const createDefaultChatHandlers = memoizeOne(
         content: string,
         options?: {
           metadata?: Record<string, string>;
-          attachmentMetadata?: AttachmentMetadata[];
+          attachmentMetadata?: FileMetadata[];
```

Comment:
does it still make sense to have the prop named "attachmentMetadata" when its type has been changed to FileMetadata?

Reply:
> does it still make sense to have the prop named "attachmentMetadata" when its type has been changed to FileMetadata?

As you mentioned in the above comment, we might introduce other attachment types in the future, like attachmentMetadata?: FileMetadata[] | InlineImageMetadata. So maybe leave it as attachmentMetadata makes sense?

Reply:
i disagree, the name should reflect the underlaying type and be consistent.